about.html

```diff
@@ -0,0 +1,142 @@
+            </div>
+        </section>
+
+        <section class="mentors">
+            <h1>Web Hero Developer Code Camp 2023 Logo</h1>
+            <div class="line"></div>
+            <p>The logo of Web Hero Fullstack Web Developer Code Camp 2023 was decided through the logo competition from
+                20, Jan to 01 Feb</p>
+            <p class="loremtxt">Lorem ipsum dolor, sit amet consectetur adipisicing elit. Ut ullam exercitationem
+                deserunt corporis dignissimos sequi dolorum! Voluptatem, exercitationem facilis? Blanditiis sunt
+                accusantium quisquam maxime odio veniam mollitia doloribus eum. Aliquid laborum ipsum eum ullam
+                perferendis quidem modi tenetur dignissimos numquam? Cumque odio aut rerum impedit aliquam molestias
+                distinctio iure pariatur.</p>
+            <img src="assets/alogo/whlogo.png" alt="">
+        </section>
+
+        <section class="mentors past-camps">
+            <h1>See the past Web Hero Camps</h1>
+            <div class="line"></div>
+            <p>Take a look at the past Web Hero Fullstack Web Developer Code Camps which took place in 2021 and 2022
+                virtually.</p>
+            <div class="camps">
+                <div class="camp-card1"></div>
+                <div class="camp-card2"></div>
+            </div>
+
+        </section>
+
+        <section class="partner" id="partners">
+            <h1>Partners</h1>
+            <div class="line"></div>
+            <div class="container">
+                <div class="partner-all">
+                    <img src="assets/socials/mozilla.svg" alt="mozilla brand name">
+                    <img src="assets/socials/google.svg" alt="google brand name">
+                    <img src="assets/socials/naver.svg" alt="naver brand name">
+                    <img src="assets/socials/daum.svg" alt="daum brand name">
+                    <img src="assets/socials/airbnb.svg" alt="airbnb brand name">
+                </div>
+
+                <div class="partner1">
+                    <img src="assets/socials/mozilla.svg" alt="mozilla brand name">
+                    <img src="assets/socials/google.svg" alt="google brand name">
+                    <img src="assets/socials/naver.svg" alt="naver brand name">
+                </div>
+                <div class="partner2">
+                    <img src="assets/socials/daum.svg" alt="daum brand name">
+                    <img src="assets/socials/airbnb.svg" alt="airbnb brand name">
+                </div>
+            </div>
+        </section>
+        <section class="license">
+            <div class="container">
+                <img src="assets/cck-wb.png" alt="license image">
+                <div class="license-info">
+                    <p>@ 2023 Creative Commons Korea. Some Rights Reserved</p>
+                    <p class="korian-lan">크리에이티브 커먼즈 코리아. 일부 권리 보유</p>
+                </div>
+            </div>
+        </section>
```

Comment:
- [ ] Your project has been styled very well, and it looks great! :+1: :100: However, to match Cindy Shin's design, it is necessary to modify the footer of the "about.html" page so that it has a dark background. Additionally, the "partners" section should be removed from the page entirely. These changes will help ensure that the website is consistent with the design vision provided by Cindy Shin, and will enhance the overall aesthetic appeal of the page. By implementing these modifications, the user experience will be improved, and the website will look more professional and visually cohesive.

**Your design**

![Screenshot from 2023-04-29 09-17-34](https://user-images.githubusercontent.com/105117832/235307802-6b9a2d86-c98b-40cd-bfd8-7b8e661a4516.png)

**Template**

![Screenshot from 2023-04-29 09-18-06](https://user-images.githubusercontent.com/105117832/235307810-ade3b02e-1d26-43c2-8b35-8b24955ea52b.png)